docs/pubsub-usage.rst

```diff
@@ -272,3 +272,30 @@ Fetch messages for a pull subscription without blocking (none pending):
    >>> messages = [recv[1] for recv in received]
    >>> [message.message_id for message in messages]
    []
+
+Fetch pending messages, acknowledging those whose processing doesn't raise an
```

Comment:
The relative pronoun ``whose`` applies for both animate and inanimate antecedents (there is no other word which could be used there).